cranelift-bforest/src/pool.rs

```diff
@@ -94,7 +94,7 @@ impl<F: Forest> NodePool<F> {
             assert!(size > 0, "Root must have more than one sub-tree");
         }
 
-        let mut done = SparseSet::new();
+        let mut done = EntitySet::new();
```

Comment:
We already know the amount of nodes at this point, so maybe `.resize` it before doing anything with it, to prevent having to copy all data over when it is resized inside the `.insert()` call.